pkg/service/webhook/storage.go

```diff
@@ -18,18 +23,54 @@ func NewWebhookStorage(db storage.ServiceStorage) (*Storage, error) {
 	return &Storage{db: db}, nil
 }
 
-func (whs *Storage) StoreWebhook(ctx context.Context, webhook Webhook) error {
-	return nil
+func (whs *Storage) StoreWebhook(ctx context.Context, noun, verb string, webhook Webhook) error {
+	storedWebhookBytes, err := json.Marshal(webhook)
+	if err != nil {
+		return util.LoggingErrorMsg(err, "webhook marshal")
+	}
+	return whs.db.Write(ctx, webhookNamespace, getWebhookKey(noun, verb), storedWebhookBytes)
 }
 
-func (whs *Storage) GetWebhook(ctx context.Context, id string) (*Webhook, error) {
-	return nil, nil
+func (whs *Storage) GetWebhook(ctx context.Context, noun, verb string) (*Webhook, error) {
+	webhookBytes, err := whs.db.Read(ctx, webhookNamespace, getWebhookKey(noun, verb))
+	if err != nil {
+		return nil, util.LoggingErrorMsg(err, "read db")
+	}
+
+	if webhookBytes == nil || len(webhookBytes) == 0 {
```

Comment:
it might be better to do the unmarshal first, and define an `isEmpty` method on the webhook struct which you call after